src/users/utils/userUtils.ts

```diff
@@ -0,0 +1,51 @@
+import { InjectRepository } from "@nestjs/typeorm";
+import { getRepository, Repository } from "typeorm";
+import { User } from "../entity/user";
+import { Event } from "../../events/entity/event";
+import { Pivot } from "../../events/entity/pivot";
+import { SubCriteria } from "../../events/entity/subCriteria";
+
+export default class UserUtils {
+  @InjectRepository(Event)
+  static eventsRepository: Repository<Event>;
+
+  @InjectRepository(User)
+  static usersRepository: Repository<User>;
+
+  static async getUserRating(Id: number) {
+    const usersRating = await getRepository(Pivot)
+      .createQueryBuilder()
+      .where({ eventId: Id })
+      .select(["userId, pivot.id"])
+      .addSelect(["COUNT(userId) AS rating"]) //distinct
+      .leftJoin(User, "user", "pivot.userId = user.id")
+      .leftJoin(
+        SubCriteria,
+        "subCriteria",
+        "subCriteria.id = pivot.subCriteriaId"
+      )
+      //  .loadRelationCountAndMap("rating", "subCriteria")
```

Comment:
We can't have db queries inside utilities, they're kind of helpers they should contain simple functions